app/deck/manage-gateway/diff.md

```diff
@@ -0,0 +1,50 @@
+---
+title: diff
```

Comment:
Here and for other titles in the Managing Kong Gateway section, I'd have a title that tells a new user what this is doing.
```suggestion
title: Show differences in configuration (`deck gateway diff`)
```